tests/dojo_tests.py

```diff
@@ -6,17 +6,35 @@ def test_create_room_successfully(self):
         ndojo = Dojo()
         initial_room_count = len(ndojo.all_rooms)
         self.assertEqual(initial_room_count,0)
-        blue_office = ndojo.create_room("Blue","office")
+        blue_office = ndojo.create_room(["Blue"],"office")
         self.assertTrue(blue_office)
         new_room_count = len(ndojo.all_rooms)
         self.assertEqual(new_room_count - initial_room_count,1)
 
     def test_create_multiple_rooms_successfully(self):
         ndojo = Dojo()
         initial_room_count = len(ndojo.all_rooms)
-        ndojo.create_room(["Blue", "Green", "Purple"], "office")
+        log = ndojo.create_room(["Blue", "Green", "Purple"], "office")
+        self.assertTrue(log)
```

Comment:
Instead of checking if variable `log` is `True`, wouldn't it be better to check compare if the variable `log` has a certain expected value?